metatron/src/main/java/uk/gov/ida/eidas/metatron/health/MetatronHealthCheck.java

```diff
@@ -3,9 +3,8 @@
 import com.codahale.metrics.health.HealthCheck;
 
 public class MetatronHealthCheck extends HealthCheck {
-
     @Override
     protected Result check() {
-        return Result.healthy("I'm completely operational, and all my circuits are functioning perfectly");
+        return Result.healthy();
```

Comment:
I guess this could be extended later to query the resolvers to see if enough of them are happy.

Reply:
There's quite a bit of cool stuff we could do here.